packages/select/src/index.mts

```diff
@@ -17,7 +17,7 @@ import chalk from 'chalk';
 import figures from 'figures';
 import ansiEscapes from 'ansi-escapes';
 
-type Choice<Value> = {
+export type Choice<Value> = {
```

Comment:
Let's revert this line (I'd prefer to not export types interfaces that end up being part of the API contract.)

```suggestion
type Choice<Value> = {
```